src/api/goods.ts

```diff
@@ -10,10 +22,10 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort(compare).splice(0, 5)); // sort and get the first 5
```

Comment:
You are using the splice method on the array returned by sort. This will alter the original array. It's better to use slice which does not modify the original array.

Reply:
done